monitor/tests/test_health.py

```
@@ -0,0 +1,178 @@
+                    'error_text': '',
+                    'meta': {}
+                }
+                
+                results = await call_health_batch(services)
+                
+                assert len(results) == 3
+                assert all(result['ok'] for result in results.values())
+                assert mock_async.call_count == 3
+        
+        # Run async test in event loop
+        import asyncio
```

Ruff (0.12.2)

Redefinition of unused `asyncio` from line 4

Remove definition: `asyncio`

(F811)